releasenotes/notes/remove-deprecated-circuit-methods-afc22d0ba59ac853.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - |
+    Remove the deprecated ``QuantumCircuit.mirror`` method and drop support for 
+    ``scale`` being the first argument in ``QuantumCircuit.draw``.
```

Comment:
```suggestion
  - |
    The previously deprecated ``mirror()`` method for :class:`qiskit.circuit.QuantumCircuit` has been removed.
    It was deprecated in the 0.15.0 release. The :meth:`qiskit.circuit.QuantumCircuit.reverse_ops` method
    should be used instead since mirroring could be confused with swapping the output qubits of the circuit.
    The ``reverse_ops()`` method only reverses the order of gates that are applied instead of mirroring.
   - |
     The previously deprecated support for a float ``scale`` value being the positional first argument in
     :meth:`qiskit.circuit.QuantumCircuit.draw` has been removed. It was deprecated in the 0.12.0 release.
     Instead to set the scale you can use the ``scale`` as a keyword argument (ie ``scale=0.75``).
```